docs/details/widgets/buttonmatrix.rst

```diff
@@ -4,44 +4,49 @@
 Button Matrix (lv_buttonmatrix)
 ===============================
 
+
 Overview
 ********
 
 The Button Matrix Widget is a lightweight way to display multiple
-buttons in rows and columns. Lightweight because the buttons are not
-actually created but just virtually drawn on the fly. This way, one
-button use only eight extra bytes of memory instead of the ~100-150
+Buttons in rows and columns --- lightweight because the buttons are not
```

Comment:
`---` how is it rendered in the docs?

Reply:
> `---` how is it rendered in the docs?

`---` is translated to an "em dash" ("—").  Based on the original wording, that was intended meaning, but ending a sentence with a period and beginning a new sentence is not the correct (English grammar) way to carry that meaning — the em-dash is the correct way to carry it.  In this context, it provides a "pause", but does not end the sentence (or the flow of thought).